prep-html-skeleton/index.html

```diff
@@ -0,0 +1,7 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <title>HTML Skeleton Exercise</title>
+    <h1>I built a proper HTML skeleton!</h1>
```

Comment:
Be sure to wrap the visible content of the page in `<body></body>` tags. The `<head></head>` of an HTML document is generally reserved for meta information (such as the document title). In the end, you will have the `head` and `body` as siblings to one another.

```html
<head>
  <!-- meta info hear -->
</head>
<body>
  <!-- visible content here -->
</body>
```

Reply:
![Screen Shot 2021-11-08 at 7 19 12 PM](https://user-images.githubusercontent.com/93749262/140856093-14076282-2ca1-4ef5-ab20-b083b2b16a0f.png)